core/store/ledgerstore/ledger_store.go

```diff
@@ -147,7 +147,7 @@ func (this *LedgerStoreImp) InitLedgerStoreWithGenesisBlock(genesisBlock *types.
 	} else {
 		genesisHash := genesisBlock.Hash()
 		exist, err := this.blockStore.ContainBlock(genesisHash)
-		if err != nil {
+		if err != nil && err != scom.ErrNotFound {
```

Comment:
this may not needed, since the ErrNotFound case has been handled in ContainBlock.